core/domain/opportunity_services_test.py

```diff
@@ -860,8 +860,10 @@ def _mock_logging_function(msg, *args):
 
         self.assertEqual(len(observed_log_messages), 0)
 
-        with self.swap(logging, 'info', _mock_logging_function), self.swap(
-            constants, 'SUPPORTED_AUDIO_LANGUAGES', mock_supported_languages):
+        with self.swap(
+            logging_services, 'info', _mock_logging_function), self.swap(
+                constants, 'SUPPORTED_AUDIO_LANGUAGES',
+                mock_supported_languages):
```

Comment:
Please separate these swaps into variables and then use the variables here.

Reply:
Done.